packages/js/src/Modules/Verto/BrowserSession.ts

```diff
@@ -63,14 +77,17 @@ export default abstract class BrowserSession extends BaseSession {
   }
 
   async connect(): Promise<void> {
-    this.sessionid = await localStorage.getItem(SESSION_ID);
+    this.sessionid = await sessionStorage.getItem(SESSION_ID);
```

Comment:
The change was made here.  
The rest is ESLint.

Reply:
Could we run ESLint on the whole repository?